docs/DeveloperGuide.md

```diff
@@ -1,3 +1,1140 @@
+        > through the list to find the Patient that matches the input
+        > specified by the user.
+
+4.  Notifying user that Patient has been found
+
+    a.  After successfully finding the Patient.
+        > PatientList.findPatient() calls PatientUI.printPatientList()
+        > to print a message notifying the user that a patient has been
+        > found, showing the patients details.
+
+**Help**
+
+help
+
+This feature shows a list of commands usable within the patient-related
+functions and their correct syntaxes that must be followed to enable
+proper usage.
+
+[Implementation:]{.ul}
+
+When a user attempts to add a patient the following classes will be
+accessed: PatientCommandInstance, UI, PatientUI, PatientParser.
+
+1.  User Executes command help
+
+    a.  PatientCommandInstance calls UI.scanInput() to read the user
+        > input
+
+    b.  PatientCommandInstance calls PatientParser.patientParse() to
+        > parse the user\'s input into a string array.
+
+2.  Checking user input in parser
+
+    a.  Using the case statements in patientParse, the command find is
+        > identified.
+
+    b.  PatientParse then calls lengthCheck() to scan the user input for
+        > any errors.
+
+    c.  If lengthCheck() throws an exception, the current action will be
+        > aborted and the user will be notified with an appropriate
+        > error message and once again prompted to enter another
+        > command. If successful, the program will move to step 3.
+
+3.  Printing the help list for the user to reference
+
+    a.  patientParse calls PatientUI.printPatientHelpList() to print a
+        > list of commands and instructions on how to use them.
+
+**Return to main menu**
+
+return
+
+This feature returns the user to the main menu of the program.
+
+[Implementation:]{.ul}
+
+When a user attempts to add a patient the following classes will be
+accessed: PatientCommandInstance, UI, PatientUI, PatientParser.
+
+1.  User Executes command return
+
+    a.  PatientCommandInstance calls UI.scanInput() to read the user
+        > input
+
+    b.  PatientCommandInstance calls PatientParser.patientParse() to
+        > parse the user\'s input into a string array.
+
+2.  Checking user input in parser
+
+    a.  Using the case statements in patientParse, the command return is
+        > identified.
+
+    b.  PatientParse then calls lengthCheck() to scan the user input for
+        > any errors.
+
+    c.  If lengthCheck() throws an exception, the current action will be
+        > aborted and the user will be notified with an appropriate
+        > error message and once again prompted to enter another
+        > command. If successful, the program will move to step 3.
+
+3.  Notifying user of returning to main menu
+
+    a.  patientParse calls UI.returningToStartMenuMessage() to show the
+        > user that they will now exit from the PatientCommandInstance.
+
+4.  Returning to main menu
+
+```{=html}
+<!-- -->
```

Comment:
![image](https://user-images.githubusercontent.com/11938072/113230174-10a1fd00-92cb-11eb-8e55-3fe537b162a0.png)
Consider removing the html comment tag.